input/pagecontent/usage.md

```diff
@@ -2,6 +2,11 @@
 Natality: When the child's name is not chosen, a dataAbsentReason should be provided with code "temp-unknown."
 <br/>Fetal Death: When the decedent fetus is not named, a dataAbsentReason should be provided with code "unknown."
 
+### Mother or Infant Transferred 
+If mother transferred for delivery, hospitalization.admitSource in the EncounterMaternity profile should be provided with code "hosp-trans." Any other admitSource code will be interpreted as 'N', with "other" being the recommended code to express mother did not transfer. The absence of a code will be interpreted as Blank (NCHS may interpret as Unknown). If source of transfer is unknown, hospitalization.admitSource should be set to "hosp-trans" and hospitalization.origin.name should be set to "UNKNOWN" in the EncounterMaternity profile.
+
+Similarly, if the infant transferred within 24 hours of delivery, hospitalization.dischargeDisposition in the EncounterBirth profile should be provided with code "other-hcf." Any other dischargeDisposition code will be interpreted as 'N', with "oth" being the recommended code to express infant did not transfer. The absence of a code will be interpreted as Blank (NCHS may interpret as Unknown). If destination of transfer is unknown, hospitalization.destination.name should be set to "UNKNOWN" in the EncounterBirth profile.
```

Comment:
Both should mention the code system, and the profiles ^shorts should be updated to explicitlly reference the codesystems (see the jira ticket for the codesystem URIs).   The binding for those fields is preferred, so I'm not sure there is much point in formally changing the binding, as long as the rules for Y/N/U are specified.